### ISSUE TITLE
Accessibility issue for fd-step-input

### ISSUE THREAD

**Issue by MIAeh** [SAP/fundamental-ngx#10122]
#### Is this a bug, enhancement, or feature request?
- Bug

#### Briefly describe your proposal.

```
 <input aria-roledescription="Step Input" aria-required="false" aria-describedby="" aria-valuemin="1" aria-valuenow="1" placeholder="" title="" id="quantity-input" name="undefined" class="fd-input fd-input--no-number-spinner fd-step-input__input" inputmode="numeric" type="text">
```
For a `fd-step-input`, the `increment` button's accessibility properties may have some issue.

![image](https://github.com/SAP/fundamental-ngx/assets/26197222/1d7af3b8-83db-41c2-b942-f5fc85f0fb57)

**Expected Behavior:**
Documents that use markup languages must be well-formed. When documents are not well-formed user agents and/or assistive technology may not correctly parse and render the content to users. Many major user-agents are able to cope well with "bad" markup, however, to be consistent across all platforms and browsers well-formed markup must be used. This is especially important for assistive technology which may or may not be programmed to perform the same level of "coping" that can be found in browsers (which are created by multinational corporations) or assistive technology brands that cost several hundred dollars or more.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
`0.36.3-rc.4`

#### If this is a bug, please provide steps for reproducing it.
Using the `fd-step-input` when checking the accessibility properties, it will happen.

**Comment by mikerodonnell89** [comment]
Closing due to inactivity and illegible screenshot/lack of details. If you're still having trouble, please open a new issue with additional details about the specific problem with the increment button
